packages/angular/src/generators/ngrx/lib/add-imports-to-module.ts

```diff
@@ -154,6 +155,20 @@ export function addImportsToModule(
 
   const isParentStandalone = !sourceText.includes('@NgModule');
 
+  const installedNgRxVersion = getInstalledPackageVersionInfo(
+    tree,
+    '@ngrx/store'
+  );
+  if (
+    installedNgRxVersion &&
+    installedNgRxVersion.major === 14 &&
+    isParentStandalone
+  ) {
+    throw new Error(
+      'With Angular 14 and NgRx 14, you must provide a path to an NgModule for property --parent.'
+    );
+  }
```

Comment:
A check for this is already being done in https://github.com/nrwl/nx/blob/master/packages/angular/src/generators/ngrx/lib/validate-options.ts#L21-L40. Could you please make sure the logic is correct there and remove this block?